lib/OpusVL/FB11/Schema/FB11AuthDB/Result/UsersData.pm

```diff
@@ -1,7 +1,7 @@
 package OpusVL::FB11::Schema::FB11AuthDB::Result::UsersData;
 
-# Created by DBIx::Class::Schema::Loader
-# DO NOT MODIFY THE FIRST PART OF THIS FILE
+# ABSTRACT: DEPRECATED - Another deprecated way of adding data to users
+our $VERSION = '0';
```

Comment:
What's with VERSION='0'?

Reply:
Just a placeholder. I don't really care to remember what version we're currently on - the file will be updated upon release.

I should confirm this assumption.

Reply:
Yep, the build dir has the correct version in it